client/app/scripts/superdesk-authoring/views/header-info.html

```diff
@@ -0,0 +1,136 @@
+<button class="proofread-toggle btn btn-light" ng-click="proofread = !proofread">
+    <span class="button-lable" translate>Mode: </span>
+    <span ng-show="proofread" translate>Proofread</span>
+    <span ng-show="!proofread" translate>Normal</span>
+</button>
+
+<span sd-theme-select ng-if="proofread" data-key="proofreadTheme"></span>
+<span sd-theme-select ng-if="!proofread" data-key="theme"></span>
+
+<ul class="metadata-list" ng-controller="MetadataWidgetCtrl" ng-if="loaded">
+    <li class="date-info full-width clearfix">
+        <label translate>Created&nbsp;</label>
+        <time datetime="{{ item.firstcreated }}" title="{{ item.firstcreated|date:'medium' }}" ng-if="item.firstcreated">{{ item.firstcreated|reldate }}&nbsp;</time>
+        <label translate>by&nbsp;</label>
+        <b>{{userLookup[item.original_creator].display_name}}&nbsp;</b>
+        <div class="modified" ng-if="item.versioncreated">
+            <label translate>Modified&nbsp;</label>
+            <time datetime="{{ item.versioncreated}}" title="{{ item.versioncreated|date:'medium' }}">{{ item.versioncreated|reldate }}</time>
+        </div>
+    </li>
+
+    <li class="full-width general-info">
+        <div class="item info-icons">
+            <i class="filetype-icon-{{item.type}}" ng-hide="hover || item.selected"></i>
+            
+            <div class="slider-icon pull-left">
+                <div class="dropdown dropleft" dropdown>
+                    <i class="urgency-icon-{{item.urgency}} dropdown-toggle" dropdown-toggle></i>
+                    <div class="dropdown-menu">
+                        <label>{{ :: "URGENCY" | translate }}</label>
+                        <div class="item">
+                            <label><i class="urgency-icon-{{item.urgency}}"></i></label>
+                            <div class="data">
+                                <div sd-meta-slider 
+                                     data-list="metadata.newsvalue" 
+                                     data-item="item" 
+                                     data-field="urgency" 
+                                     data-change="autosave(item)" 
+                                     ng-disabled="!_editable">
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="slider-icon pull-left" ng-if="item.urgency">
+                <div class="dropdown dropleft" dropdown>
+                    <i class="output-item-label label-{{item.urgency}} dropdown-toggle" dropdown-toggle>{{ item.urgency }}</i>
+                    <div class="dropdown-menu">
+                        <label>{{ :: "NEWS VALUE" | translate }}</label>

```

Comment:
Yeah, I know, but I see it like this on wiki page (https://wiki.sourcefabric.org/display/NR/AE+-+UI+changes?preview=/32571452/32571824/AE_screen__header_info_new_4a.png) and I asked Vladimir. He told me to add both fields, urgency and news value, but with different icons and to allow both fields to be editable.